### PR TITLE
Improve preview of the site

### PR DESCRIPTION
**Pull request by sweatC**
https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fwww.jetthoughts.com
https://developers.facebook.com/tools/debug/sharing/?q=jetthoughts.com
https://cards-dev.twitter.com/validator
Need use image with proportions 1x1

### CODE REVIEW

**Comment by sweatC** [comment]
image
image

**Comment by pftg** [comment]
@sweatC I could not find text with title and desc in preview. We should see title, desc and logo

Please fill all fields from https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary

For discrod, we should have but with square logo instead of landscape:

**Comment by andriyParashchuk** [comment]
@sweatC Need replace `content="summary_large_image"` to`content="summary"` or other type

**Comment by sweatC** [comment]
image

**Comment by ad-sys** [comment]
Just checked in discord, looks cool to me

**Comment by pftg** [comment]
@sweatC

**Comment by pftg** [comment]
Linkedin shows bad data: 

also bad image - looks poorly 

Some ideas on how to improve:

**Comment by pftg** [comment]
Facebook:

**Comment by pftg** [comment]
@sweatC @ad-sys for verification please use links which have been provided and messengers: Skype, Telegram, Slack, and Discord

**Comment by ad-sys** [comment]
Oh, okay, thanks for pointing it out, will do!

**Comment by sweatC** [comment]
1. Could not use SVG in OG protocol. Need to think about how to get a better picture(png is a good format for this).
2. We already have `og:type` meta tag, see: https://github.com/jetthoughts/jetthoughts.github.io/blob/392-change-site-preview/_includes/head.html#L16.
3. Added `og:image:alt` attribute.

File: _includes/head.html [hunk 1]
```diff
@@ -20,6 +20,7 @@
   <meta property="og:description" content="{{ site.description }}">
   <meta property="og:image" content="{{ site.url }}/logo_400.jpeg">
   <meta property="og:image:secure_url" content="{{ site.url }}/logo_400.jpeg">
+  <meta property="og:image:alt" content="Company logo. Big white circle on blue background">
```
Comment (pftg) [inline _includes/head.html]: I do not think this is the best practice to have such text for alt. could you research what people add for this attribut

Comment (sweatC) [reply _includes/head.html]: A description of what is in the image (not a caption).

Comment (sweatC) [reply _includes/head.html]: It just like a normal alt. So, what is your proposition for this text?
image

Comment (pftg) [reply _includes/head.html]: `JetThoughts logo: white triskelion in the circle on the dark blue background` - maybe

**Comment by pftg** [comment]
@sweatC please check alt text. Also please confirm that li and other validators are green. Please post screenshots with their results. As I got LI has not been changed, but expected to see updates instead of

**Comment by ad-sys** [comment]
http://guteurls.de/
https://metatags.io/
Пытался найти сервис чтобы предусмотреть все варианты пока нашел только эти

**Comment by pftg** [comment]
- [ ] LinkedIn:

File: _includes/head.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
   <meta property="og:description" content="{{ site.description }}">
   <meta property="og:image" content="{{ site.url }}/logo_500.png">
   <meta property="og:image:secure_url" content="{{ site.url }}/logo_500.png">
-  <meta property="og:image:alt" content="Company logo. Big white circle on blue background">
+  <meta property="og:image:alt" content="JetThoughts logo: white triskelion in the circle on the dark blue background">
```
Comment (pftg) [inline _includes/head.html:25]: 👍

**Comment by andriyParashchuk** [comment]
@pftg Linkedin shows the only title. We can't show description for linkedin

**Comment by andriyParashchuk** [comment]
@sweatC Please remove the double logo images

**Comment by andriyParashchuk** [comment]
@sweatC 
image

Need add tags:
`<meta name="pubdate" content="20180101">`
`<meta name="author" content="Jetthoughts">`

**Comment by andriyParashchuk** [comment]
image
https://github.com/gohugoio/hugo/issues/3095
Need add tag:
`<meta name="twitter:text:title" content="{{ page.title }}"/>`

**Comment by pftg** [comment]
> @sweatC
> image
> 
> Need add tags:
> `<meta name="pubdate" content="20180101">`
> `<meta name="author" content="Jetthoughts">`

I do not think, I'd better switched to another type instead of an article. What are the options?

**Comment by andriyParashchuk** [comment]
@pftg  It's strange because we use type `website`
https://github.com/jetthoughts/jetthoughts.github.io/blob/fa9acfeb4f4c8143185fc26ac9dc22e3deea9212/_includes/head.html#L16

Schema.org also dos not have type `article`
image

**Comment by pftg** [comment]
@andriyParashchuk I think this is not valid value. You could check: http://ogp.me/#object-type

**Review by andriyParashchuk — COMMENTED** [review]
LGTM

**Comment by pftg** [comment]
@andriyParashchuk please merge if you confirm.

**Comment by pftg** [comment]
@andriyParashchuk @sweatC

**Comment by pftg** [comment]
> @sweatC Please remove the double logo images

have this been fixed?

**Comment by sweatC** [comment]
Yes.

**Comment by sweatC** [comment]
@pftg to prev comment 
image

**Comment by pftg** [comment]
Have not hard refreshed my page! Thanks for update!

**Pull request merged**